ggml/src/ggml-quants.h

```diff
@@ -127,6 +127,10 @@ void iq2xs_free_impl(enum ggml_type type);
 void iq3xs_init_impl(int grid_size);
 void iq3xs_free_impl(int grid_size);
 
+#if defined(__ARM_FEATURE_SVE)
+extern int sve_cnt_b;
```

Comment:
I think it would be good to add a `ggml_` prefix to this variable to avoid possible conflicts with 3rd party software.
```suggestion
extern int ggml_sve_cnt_b;
```

Reply:
ok, yes, that makes sense. i changed it according to your suggestion